sources/Cargo.toml

```diff
@@ -68,6 +68,7 @@ members = [
     "api/migration/migrations/v1.9.0/kernel-modules-setting",
     "api/migration/migrations/v1.9.0/kernel-modules-setting-metadata",
     "api/migration/migrations/v1.9.0/kubelet-no-daemon-reload",
+    "api/migration/migrations/v1.10.0/add-eks-autoscaling",
     "api/migration/migrations/v1.9.0/updates-targets-base-url",
```

Comment:
This migration should be ordered below the ones for `1.9.0`

Reply:
I have moved that migration below the 1.9.0 migrations and shifted the change to ["Handle migrations for eks-autoscaling.should-wait"](https://github.com/bottlerocket-os/bottlerocket/pull/2322/commits/c173ed58cc568c2664e44db6fd36520b347aa5be)